packages/pluggableWidgets/accordion-web/src/components/AccordionGroup.tsx

```diff
@@ -89,6 +91,13 @@ export function AccordionGroup(props: AccordionGroupProps): ReactElement | null
         }
     }, [props.collapsed, props.visible, previousCollapsedValue, animateContent]);
 
+    useEffect(() => {
+        if (previousCollapsedValue !== previousPreviousCollapsedValue.current) {
+            previousPreviousCollapsedValue.current = previousCollapsedValue;
+            onToggleCompletion?.(props.collapsed);
+        }
+    }, [props.collapsed, onToggleCompletion, previousCollapsedValue, previousPreviousCollapsedValue]);
```

Comment:
previousPreviousCollapsedValue could be `previousPreviousCollapsedValue.current`

Reply:
Renamed the refs a bit, because I didn't like the naming of them in AccordionGroup. I removed the ref from the dep array, because it's not necessary for refs. This also aligns with the other dep arrays of hooks for AccordionGroup.